tests/components/flick_electric/snapshots/test_diagnostics.ambr

```diff
@@ -0,0 +1,15 @@
+# serializer version: 1
+# name: test_diagnostics
+  dict({
+    'data': dict({
+      '__type': "<class 'pyflick.types.FlickPrice'>",
+      'repr': "FlickPrice({'id': '_2025-02-09 05:30:00 UTC..2025-02-09 05:59:59 UTC', 'type': 'rating_rated_period', 'start_at': '2025-02-09T05:30:00.000Z', 'end_at': '2025-02-09T05:59:59.000Z', 'status': 'final', 'cost': '0.20011', 'import_cost': '0.20011', 'export_cost': None, 'cost_unit': 'NZD', 'quantity': '1.0', 'import_quantity': '1.0', 'export_quantity': None, 'quantity_unit': 'kwh', 'renewable_quantity': None, 'generation_price_contract': None, 'components': [{'id': '213507464_1_kwh_generation_UN_24_default_2025-02-09 05:30:00 UTC..2025-02-09 05:59:59 UTC', 'type': 'rating_component', 'charge_method': 'kwh', 'charge_setter': 'generation', 'value': '0.20011', 'quantity': '1.0', 'unit_code': 'NZD', 'charge_per': 'kwh', 'flow_direction': 'import', 'content_code': 'UN', 'hours_of_availability': 24, 'channel_number': 1, 'meter_serial_number': '213507464', 'price_name': 'default', 'applicable_periods': [], 'single_unit_price': '0.20011', 'billable': True, 'renewable_quantity': None, 'generation_price_contract': 'FLICK_FLAT_2024_04_01_midpoint'}, {'id': '213507464_1_kwh_network_UN_24_offpeak_2025-02-09 05:30:00 UTC..2025-02-09 05:59:59 UTC', 'type': 'rating_component', 'charge_method': 'kwh', 'charge_setter': 'network', 'value': '0.0406', 'quantity': '1.0', 'unit_code': 'NZD', 'charge_per': 'kwh', 'flow_direction': 'import', 'content_code': 'UN', 'hours_of_availability': 24, 'channel_number': 1, 'meter_serial_number': '213507464', 'price_name': 'offpeak', 'applicable_periods': [], 'single_unit_price': '0.0406', 'billable': False, 'renewable_quantity': None, 'generation_price_contract': 'FLICK_FLAT_2024_04_01_midpoint'}]})",
```

Comment:
Yeah, we need to present this better, let's make sure this is a dictionary instead.